projects/vanilla/src/app/search/search.component.html

```diff
@@ -153,8 +145,17 @@
                 </ng-template>
 
                 <ng-template uib-template="my-search" display="My search" let-config>
-                    <sq-facet-mysearch [results]="results" [allowDeletion]="config.allowDeletion"></sq-facet-mysearch>
-                </ng-template>
+                  <div class="d-flex small sq-toolbar">
+                    <sq-filters-view
+                      class="flex-grow-1 position-relative"
+                      [query]="searchService.query"
+                      [closeOnClickOutside]="true"
+                      [advancedModeClass]="'card position-absolute p-3'"
+                      (filterEdit)="searchService.search()">
+                    </sq-filters-view>
+                    <sq-results-counter [rowCount]="searchService.rowCount" class="mx-2"></sq-results-counter>
+                    <sq-sort-selector [results]="results" [rightAligned]="true"></sq-sort-selector>
+                </div>                </ng-template>
```

Comment:
I think this ng-template should contain only the sq-filters-view component, as the result-counter and sort-selector are in their own ng-template.